packages/atlas/src/joystream-lib/lib.ts

```diff
@@ -91,11 +92,11 @@ export class JoystreamLib {
     return number.toNumber()
   }
 
-  async subscribeAccountBalance(accountId: AccountId, callback: (balance: number) => void) {
+  async subscribeAccountBalance(accountId: AccountId, callback: (balance: BN) => void) {
     await this.ensureApi()
 
     const unsubscribe = await this.api.derive.balances.all(accountId, ({ availableBalance }) => {
-      callback(availableBalance.toNumber())
+      callback(availableBalance)
```

Comment:
Could you also update `getAccountBalance` above? Or just delete it since I don't think it's used?